mods/default/furnace.lua

```diff
@@ -154,6 +157,9 @@ local function furnace_node_timer(pos, elapsed)
 					else
 						dst_full = true
 					end
+					-- Play cooling sound
+					minetest.sound_play("default_cool_lava",
+						{pos = pos, max_hear_distance = 16, gain = 0.1})
```

Comment:
```suggestion
						{pos = pos, max_hear_distance = 16, gain = 0.1}, true)
```
same for the other sound_play call

Reply:
Done

Reply:
What about making the second sound a bit quieter?

Reply:
Maybe a little. Shall I change the audio file or the `gain` variable?

Reply:
latter IMO

Reply:
Agreed; that would be non-destructive, compared to editing the audio.

Reply:
By the way, which sound are you referring to? I suppose you mean the cooling sound?

Reply:
`default_furnace_active`

Reply:
Eek, I forgot what `gain` actually does xD

Okay, I've reduced it to `0.5`.